pkg/network/protocols/http2/protocol.go

```diff
@@ -244,6 +256,13 @@ func (p *protocol) Stop(_ *manager.Manager) {
 	if p.statkeeper != nil {
 		p.statkeeper.Close()
 	}
+
+	if p.telemetry != nil {
+		p.stopOnce.Do(func() {
```

Comment:
no need to have stopOne here as the teardown of the protocol is called only once